core/mod/mod.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 The KCL Authors. All rights reserved.
+
+// Package mod is the core abstraction in kpm for working with a package or module of kcl.
+package mod
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/BurntSushi/toml"
+	conf "kusionstack.io/kpm/core/conf"
+	"kusionstack.io/kpm/core/reporter"
+	"kusionstack.io/kpm/gen/pkg"
+)
+
+const kclMod = "kcl.mod"
+
+// KclPkg is the package of kcl.
+type KclPkg struct {
+	HomePath string       `toml:"-"`
+	Pkg      *pkg.Package `toml:"package"`
+}
+
+// NewKclPkg new a kcl package `KclPkg`` based on `Config`.
+func NewKclPkg(conf conf.Config) KclPkg {
```

Comment:
rename to `func InitPackage(conf *conf.Config) (*Package, error)`

Reply:
fixed.